engine-cdi/compatibility-test-weld2/pom.xml

```
@@ -0,0 +1,154 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <name>Camunda Platform - engine - Cdi - Compatbility Test Weld 2</name>
+  <artifactId>camunda-engine-cdi-compatibility-test-weld2</artifactId>
+  <packaging>jar</packaging>
```

```suggestion
  <packaging>jar</packaging>
```
Why do we need to build a JAR for that module?

---

Answer: When packaging "pom", the `test-compile` and `test` phases are note executed.
https://maven.apache.org/guides/introduction/introduction-to-the-lifecycle.html#default-lifecycle-bindings-packaging-pom